### ISSUE TITLE
README warning with NPM

### ISSUE THREAD

**Issue by ooflorent** [MoOx/eslint-loader#47]
```sh
$ npm i
npm WARN package.json eslint-loader@0.14.0 No README data
```

**Comment by bebraw** [comment]
Looks like `package.json` `files` field is missing `README.md`. That probably explains it as `README.md` won't be included to the distribution version in this case. Easy to fix.

**Comment by MoOx** [comment]
@bebraw README is inlined in package.json during the `npm publish` task. I do the same for lots of other module without this issue. I guess something weird happened.

**Comment by bebraw** [comment]
@MoOx Ok, cool!

**Comment by MoOx** [comment]
I just released 0.14.1 (no special attention to this issue) I hope it's working now.

```console
$ npm i -D eslint-loader
eslint-loader@0.14.1 node_modules/eslint-loader
└── loader-utils@0.2.10 (big.js@3.1.3, json5@0.4.0)

```
